ui/src/components/system/UsersGroups.vue

```diff
@@ -1535,15 +1535,15 @@
                           </div>
                         </div>-->
                         <div
-                          v-if="newProvider.isChecked && k!='action' && k != 'NsdcIp' && k!='DiscoverDcType' && k!='port' && k!='IsLocal' && k!='isAD' && k!='isLdap' && k!='host' && k!='Provider' && !((newProvider.info.BindType == 'anonymous' && k=='BindPassword') || (newProvider.info.BindType == 'anonymous' && k=='BindDN'))"
+                          v-if="newProvider.isChecked && k!='action' && k != 'LdapUriDn' && k != 'NsdcIp' && k!='DiscoverDcType' && k!='port' && k!='IsLocal' && k!='isAD' && k!='isLdap' && k!='host' && k!='Provider' && !((newProvider.info.BindType == 'anonymous' && k=='BindPassword') || (newProvider.info.BindType == 'anonymous' && k=='BindDN'))"
                           v-for="(v,k) in newProvider.info"
                           v-bind:key="k"
                           class="form-group"
                         >
                           <label
                             class="col-sm-3 control-label"
                             for="textInput-modal-markup"
-                          >{{k | camelToSentence}}</label>
```

Comment:
I can't understand what is the effect of this change. 
Could you please provide and emaple?

Reply:
I added a screenshot to the PR description

Reply:
I added a screenshot to the PR description